userguide/tutorials/getting_started.adoc

```diff
@@ -288,7 +288,7 @@ The next step is to configure Kill Bill. For this, you need to follow the steps
 
 . Verify that there are no errors in the Kill Bill logs on the console and that the logs display a line which states that  *Kill Bill server has started*
 
-. Open a browser and type http://localhost:8080/index.html. If Kill Bill is configured properly, it should display the following page:
+. Open a browser and type `\http://localhost:8080/index.html`. If Kill Bill is configured properly, it should display the following page:
```

Comment:
I think you should be able to specify `\http://localhost:8080/index.html` directly, without the backticks

Reply:
Done